src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -205,6 +254,33 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             (mask_sbp_adherent) & (mask_sbp_two_drugs), data_values.COLUMNS.SBP_MULTIPLIER
         ] = data_values.SBP_MULTIPLIER.TWO_DRUGS
 
+        pop[data_values.COLUMNS.LDLC_MULTIPLIER] = 1
+        mask_ldlc_adherent = (
+            pop[data_values.COLUMNS.LDLC_MEDICATION_ADHERENCE]
+            == data_values.MEDICATION_ADHERENCE_TYPE.ADHERENT
+        )
+        mask_ldlc_low = (
```

Comment:
Consider including adherence in these masks so you don't have to use it later below. It's fine either way though.